modules/wkt/src/wkt-loader.ts

```diff
@@ -15,6 +16,7 @@ export const WKTWorkerLoader: Loader = {
   mimeTypes: ['text/plain'],
   category: 'geometry',
   text: true,
+  tests: ['WKT('],
```

Comment:
Is this true? For individual geometries it wouldn't be prefaced by `WKT` would it?